v2/yarpcthrift/thriftrw-plugin-yarpc2/server.go

```diff
@@ -110,12 +110,12 @@ func (h handler) <.Name>(ctx <$context>.Context, body <$wire>.Value) (<$yarpcthr
 		err := h.impl.<.Name>(ctx, <range .Arguments>args.<.Name>,<end>)
 	<end>
 
-	hadError := err != nil
+	appErr := err
```

Comment:
nit: should we just call the error returned in the above if block `appErr` so we can avoid this line?